src/primitives.ts

```diff
@@ -0,0 +1,45 @@
+// Ideally: SeverityLevel is not an enum, see: https://github.com/microsoft/ApplicationInsights-node.js/issues/942
+export type strictAiSeverityLevel =
+  | 0
+  | 1
+  | 2
+  | 3
+  | 4
+
+export type pinoSeverityLevel =
+  | 10
+  | 20
+  | 30
+  | 40
+  | 50
+  | 60
+
+type NextLoggerNames =
+  | 'console'
+  | 'next.js'
+  | 'Error' // exception??
+
+/**
+ * Generated by pino inside of next-logger, I think
+ */
+export type LogItem = {
```

Comment:
I should re-word this I think

I was originally not all too sure where this object shape came from.

`NextLoggerNames` is also deceiving. I'm making assumptions/hard-coding against next-logger. But, this package currently doesn't have any special relationship with next-logger, and I'd hate to quietly bake-in this tiny little assumption. Besides `NextLoggerNames` type, which is not even accurate, this fork is STILL totally generic, and has no hard-coded assumptions that it's being used with next-logger. (I think)